core-bundle/src/EventListener/CommandSchedulerListener.php

```diff
@@ -52,9 +50,16 @@ private function canRunCron(Request $request): bool
             return false;
         }
 
-        $config = $this->framework->getAdapter(Config::class);
+        // Without the db table, the cron framework cannot work
+        if (!$this->canRunDbQuery()) {
+            return false;
+        }
+
+        if ($this->autoMode && !$this->cron->hasMinutelyCliCron()) {
```

Comment:
```suggestion
        if ($this->autoMode && $this->cron->hasMinutelyCliCron()) {
```
If we are in `auto` mode and we _have_ a minutely cli cron we want to skip the frontend cron, right?

Reply:
Lol, wrote the tests incorrectly. Of course. Fixed in https://github.com/contao/contao/pull/5591/commits/e2999b5ef709db746ddbddebd8b259b05940bf12.